### PR TITLE
Nft stop inconsistency

### PR DESCRIPTION
**Pull request by sasurobert**
## Reasoning behind the pull request
- 
- 
- 
  
## Proposed changes
- 
- 
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: config/tomlConfig_test.go [hunk 1]
```diff
@@ -895,12 +898,12 @@ func TestEnableEpochConfig(t *testing.T) {
 			StorageAPICostOptimizationEnableEpoch:             54,
 			TransformToMultiShardCreateEnableEpoch:            55,
 			ESDTRegisterAndSetAllRolesEnableEpoch:             56,
-			ScheduledMiniBlocksEnableEpoch:                 57,
-			CorrectJailedNotUnstakedEmptyQueueEpoch:         58,
-			DoNotReturnOldBlockInBlockchainHookEnableEpoch:            59,
-			AddFailedRelayedTxToInvalidMBsDisableEpoch:       60,
-			SCRSizeInvariantOnBuiltInResultEnableEpoch:              61,
-			CheckCorrectTokenIDForTransferRoleEnableEpoch:                   62,
+			ScheduledMiniBlocksEnableEpoch:                    57,
```
Comment (iulianpascalau) [inline config/tomlConfig_test.go:901]: :+1:

File: config/tomlConfig_test.go [hunk 2]
```diff
@@ -910,22 +913,23 @@ func TestEnableEpochConfig(t *testing.T) {
...
-			RelayedNonceFixEnableEpoch:                        80,
-			SetGuardianEnableEpoch:                            81,
-			AutoBalanceDataTriesEnableEpoch:                   82,
-			KeepExecOrderOnCreatedSCRsEnableEpoch:             83,
-			MultiClaimOnDelegationEnableEpoch:                 84,
-			ChangeUsernameEnableEpoch:                         85,
-			ConsistentTokensValuesLengthCheckEnableEpoch:      86,
-			FixDelegationChangeOwnerOnAccountEnableEpoch:      87,
-			ScToScLogEventEnableEpoch:                         88,
+			FixOldTokenLiquidityEnableEpoch:                   72, RuntimeMemStoreLimitEnableEpoch: 73,
```
Comment (iulianpascalau) [inline config/tomlConfig_test.go:916]: please add empty line after  `,`. Valid also on L872

File: integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go [hunk 3]
```diff
@@ -5,6 +5,7 @@ package esdtNFT
 import (
 	"bytes"
 	"encoding/hex"
+	logger "github.com/multiversx/mx-chain-logger-go"
```
Comment (iulianpascalau) [inline integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go:8]: go imports

File: integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go [hunk 4]
```diff
@@ -1081,3 +1082,146 @@ func TestESDTSFTWithEnhancedTransferRole(t *testing.T) {
...
+		vm.ESDTSCAddress,
+		string(txData),
+		integrationTests.AdditionalGasLimit+core.MinMetaTxExtraGasCost,
+	)
+
+	time.Sleep(time.Second)
+	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, 15, nonce, round, idxProposers)
+	time.Sleep(time.Second)
+
+	_ = logger.SetLogLevel("process/smartcontract:TRACE")
```
Comment (iulianpascalau) [inline integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go:1163]: remove SetLogLevel?

File: vm/process/systemVM.go [hunk 5]
```diff
@@ -118,12 +118,6 @@ func (s *systemVM) RunSmartContractCall(input *vmcommon.ContractCallInput) (*vmc
 		}, nil
 	}
 
-	// asyncParams, err := contexts.RemoveAsyncContextArguments(&input.VMInput)
```
Comment (iulianpascalau) [inline vm/process/systemVM.go:121]: :+1:

File: config/tomlConfig_test.go (hunk 2 above)
Comment (sasurobert) [reply config/tomlConfig_test.go:916]: done

File: integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go (hunk 3 above)
Comment (sasurobert) [reply integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go:8]: deleted logger

File: integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go (hunk 4 above)
Comment (sasurobert) [reply integrationTests/vm/esdt/nft/esdtNFT/esdtNft_test.go:1163]: done

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
Normal allin test: v1.5.13-dev-config-8c685c8e00 -> nft-stop-inconsistency-b069c1d24f

--- Specific errors ---

block hash does not match 5929
wrong nonce in block 2072
miniblocks does not match 0
num miniblocks does not match 0
miniblock hash does not match 0
block bodies does not match 0
receipts hash missmatch 2

/------/


--- Statistics ---

Nr. of all ERRORS:  0
Nr. of all WARNS:  166
Nr. of new ERRORS:  0
Nr. of new WARNS:  0
Nr. of PANICS:  0

/------/

--- ERRORS ---

/------/

--- WARNINGS ---

/------/

File: process/smartContract/processorV2/processV2.go [hunk 6]
```diff
@@ -282,7 +282,7 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		txHash := sc.computeTxHashUnsafe(tx)
 		log.Debug(fmt.Sprintf("scProcessor.ExecuteSmartContractTransaction(): execution took > %s", executeDurationAlarmThreshold), "tx hash", txHash, "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()))
 	} else {
-		log.Trace("scProcessor.ExecuteSmartContractTransaction()", "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()))
+		log.Trace("scProcessor.ExecuteSmartContractTransaction()", "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()), "retMessage", string(failureContext.returnMessage))
```
Comment (bogdan-rosianu) [inline process/smartContract/processorV2/processV2.go:285]: might log this for the long execution time as well

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**